SpatialStructure-4/readme.md

```diff
@@ -0,0 +1,165 @@
+        - Representation=IfcExtrudedAreaSolid
+        - PredefinedType*=IfcRoadPartTypeEnum.INTERSECTION
+        - UsageType=LONGITUDINAL
+      - A.a.1.8 IfcFacilityPart (Sammalammintie segment 386 - 620)
+        - Representation=IfcExtrudedAreaSolid
+        - PredefinedType*=IfcRoadPartTypeEnum.ROADSEGMENT
+        - UsageType=LONGITUDINAL
+    - A.a.2 IfcRoad (Safety equipment road)
+      - Representation=IfcExtrudedAreaSolid
+      - A.a.2.1 IfcFacilityPart (Safety equipment terminal)
+        - Representation=IfcExtrudedAreaSolid
+        - PredefinedType*=IfcFacilityPartCommonTypeEnum.TERMINAL
+        - UsageType=LONGITUDINAL
+      - A.a.2.2 IfcFacilityPart (Sammalammintie safety equipment road intersection)
+        - Representation=IfcExtrudedAreaSolid - Shared with A.a.1.4
+        - PredefinedType*=IfcRoadPartTypeEnum.INTERSECTION
+        - UsageType=LONGITUDINAL
+        - IfcRelInterferesElements
+          - InterferenceType=Crosses
+          - RelatingElement=#A.a.1.4
+          - RelatedElement=#A.a.2.2
+    - A.a.3 IfcRoad (Road2)
+      - Representation=IfcExtrudedAreaSolid
+      - A.a.3.1 IfcFacilityPart (Road2 segment 18 - 36)
+        - Representation=IfcExtrudedAreaSolid
+        - PredefinedType*=IfcRoadPartTypeEnum.ROADSEGMENT
+        - UsageType=LONGITUDINAL
+      - A.a.3.2 IfcFacilityPart (Sammalammintie Road2 intersection)
+        - Representation=IfcExtrudedAreaSolid - Shared with A.a.1.7
+        - PredefinedType*=IfcRoadPartTypeEnum.INTERSECTION
+        - UsageType=LONGITUDINAL
+        - IfcRelInterferesElements
+          - InterferenceType=Crosses
+          - RelatingElement=#A.a.1.7
+          - RelatedElement=#A.a.3.2
+    - A.a.4 IfcRailway (Railway)
+      - IfcExtrudedAreaSolid
+      - A.a.4.1 IfcFacilityPart (Railway part 1)
+        - Representation=-
+        - PredefinedType*=IfcRailwayPartTypeEnum.TRACKSTUCTUREPART
+        - UsageType=LONGITUDINAL
+      - A.a.4.2 IfcFacilityPart (Sammalamminitie level crossing)
+        - Representation=IfcExtrudedAreaSolid - Shared with A.a.1.5
+        - PredefinedType*=IfcFacilityPartCommonTypeEnum.LEVELCROSSING
+        - UsageType=LONGITUDINAL
+        - IfcRelInterferesElements
+          - InterferenceType=Crosses
+          - RelatingElement=#A.a.1.5
+          - RelatedElement=#A.a.4.2
+      - A.a.4.3 IfcFacilityPart (Railway part 3)
+        - Representation=-
+        - PredefinedType*=IfcRailwayPartTypeEnum.TRACKSTUCTUREPART
+        - UsageType=LONGITUDINAL
+      - A.a.4.4 IfcFacilityPart (Turvalaite Terminal)
+        - Representation=IfcExtrudedAreaSolid - Shared with A.1.2.1
+        - PredefinedType*=IfcFacilityPartCommonTypeEnum.TERMINAL
+        - UsageType=LONGITUDINAL
+        - IfcRelInterferesElements
+          - InterferenceType=Crosses
+          - RelatingElement=#A.a.2.1
+          - RelatedElement=#A.a.4.4
+
+_* the PredefinedType enumeration is selected through "IfcFacilityPartTypeSelect"_.
+
+
+---
```

Comment:
This will be fun seeing in the resulting PDF.

I'd suggest removing it altogether.

Reply:
Ok. You mean the loooong list?

Reply:
Removed the list altogether.

Reply:
No, I mean the horizontal line :D

Reply:
Oops...

Reply:
I quite liked the list, please reverse the content! :)

Reply:
Just to be really clear - which horizontal line do you mean?

Reply:
I mean the three `---` there.

Reply:
I hope that they are gone...

Reply:
Yes, looks good now.